src/hyperlight_host/src/sandbox/uninitialized.rs

```diff
@@ -847,7 +837,19 @@ mod tests {
     // from their workstation to be successful without needed to know about test interdependencies
     // this test will be run explicitly as a part of the CI pipeline
     #[ignore]
+    #[cfg(feature = "build-metadata")]
```

Comment:
Don't understand why this is behind the build metadata feature? its testing logging and tracing?

Reply:
This is only used by tests that are gated behind the build-metadata feature, so, without gating this, we get errors about dead_code.